tsconfig.json

```diff
@@ -1,23 +1,19 @@
 {
+  "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
-    "allowSyntheticDefaultImports": false,
-    "allowUnreachableCode": false,
-    "allowUnusedLabels": false,
-    "alwaysStrict": true,
-    "declaration": true,
-    "forceConsistentCasingInFileNames": true,
-    "lib": ["es6"],
-    "noFallthroughCasesInSwitch": true,
-    "noEmitOnError": true,
-    "noImplicitAny": true,
-    "noImplicitReturns": true,
-    "noImplicitThis": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "pretty": true,
-    "strictNullChecks": true,
-    "module": "commonjs",
-    "target": "es5",
-    "sourceMap": true
-  }
-}
+    "rootDir": ".",
+    "outDir": "build",
+    "lib": ["es2015"]
+  },
+  "include": [
+    "src/*.ts",
```

Comment:
I think `src/**/*.ts` covers this (same comment on `test/*.ts` and `system-test/*.ts`